src/components/application_manager/src/state_controller_impl.cc

```diff
@@ -159,7 +164,7 @@ void StateControllerImpl::SetRegularState(
     const mobile_apis::AudioStreamingState::eType audio_state,
     const mobile_apis::VideoStreamingState::eType video_state,
     const mobile_apis::SystemContext::eType system_context,
-    const bool send_activate_app) {
+    const bool send_state_to_hmi) {
```

Comment:
@jacobkeeler confusing naming send_state_to_hmi. 
This boolean parameter controls if request to HMI should be used for put application in the new state. 
`send_state_to_hmi` sounds like you will notify HMI about changing state. I would suggest to rename it to "request_hmi"

Reply:
@LuxoftAKutsan fixed with https://github.com/smartdevicelink/sdl_core/pull/2948/commits/8080f10bd7b7f5bbe074cbdafdc19fba7ebe8118